redis.conf

```diff
@@ -629,10 +629,8 @@ repl-diskless-sync-delay 5
 repl-diskless-sync-max-replicas 0
 
 # -----------------------------------------------------------------------------
-# WARNING: RDB diskless load is experimental. Since in this setup the replica
-# does not immediately store an RDB on disk, it may cause data loss during
-# failovers. RDB diskless load + Redis modules not handling I/O reads may also
-# cause Redis to abort in case of I/O errors during the initial synchronization
+# WARNING: RDB diskless load + Redis modules not handling I/O reads may cause
+# Redis to abort in case of I/O errors during the initial synchronization
 # stage with the master. Use only if you know what you are doing.
```

Comment:
if we dropped the "experimental" tag, maybe we should drop the last part ("only if you know what you are doing") too.
leaving the warning and disabled by default in place for now.

Reply:
Done ;)